src/services/track/helpers.ts

```diff
@@ -31,3 +34,19 @@ export function isDIDTrack(data: TrackData): data is IDIDTrack {
 export function toCoin(amount: bigint, denom = MINIMAL_DENOM): Coin {
 	return coin(amount.toString(), denom);
 }
+
+export function builSubmitOperation (subscription: Stripe.Subscription, name: string, options?: ISubmitOptions) {
```

Comment:
Marking typo to resolve:

```suggestion
export function buildSubmitOperation (subscription: Stripe.Subscription, name: string, options?: ISubmitOptions) {
```